helpdesk_mgmt_fieldservice/models/fsm_order.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class FSMOrder(models.Model):
+    _inherit = 'fsm.order'
+
+    ticket_id = fields.Many2one('helpdesk.ticket', string='Ticket',
+                                track_visibility='onchange')
+
+    def action_complete(self):
+        if not self.date_end:
+            raise ValidationError(_("Cannot move to Complete " +
+                                    "until 'Actual End' is filled in"))
+        if not self.resolution:
+            raise ValidationError(_("Cannot move to Complete " +
+                                    "until 'Resolution' is filled in"))
```

Comment:
```suggestion
```

I would suggest to remove this validation logic.  Once [Field Service 550](https://github.com/OCA/field-service/pull/550) is merged, similar validation can be achieved and is more configurable for individual use cases.